packages/e2e/test/wallet/PersonalWallet/conwayTransactions.test.ts

```diff
@@ -0,0 +1,395 @@
+      const newRegCert: Cardano.NewStakeAddressCertificate = {
+        __typename: CertificateType.Registration,
+        deposit: stakeKeyDeposit,
+        stakeCredential
+      };
+      const signedTx = await wallet
+        .createTxBuilder()
+        .customize(({ txBody }) => ({ ...txBody, certificates: [newRegCert] }))
+        .build()
+        .sign();
+      const [, confirmedTx] = await submitAndConfirm(wallet, signedTx.tx, 1);
+
+      assertTxHasCertificate(confirmedTx, newRegCert);
+      expect((await firstValueFrom(wallet.delegation.rewardAccounts$))[0].keyStatus).toBe(StakeKeyStatus.Registered);
+    });
+
+    it('can un-register stake key through the new Conway certificates', async () => {
+      const newUnRegCert: Cardano.NewStakeAddressCertificate = {
+        __typename: CertificateType.Unregistration,
+        deposit: stakeKeyDeposit,
+        stakeCredential
+      };
+      const signedTx = await wallet
+        .createTxBuilder()
+        .customize(({ txBody }) => ({ ...txBody, certificates: [newUnRegCert] }))
+        .build()
+        .sign();
+      const [, confirmedTx] = await submitAndConfirm(wallet, signedTx.tx, 1);
+
+      assertTxHasCertificate(confirmedTx, newUnRegCert);
+      expect((await firstValueFrom(wallet.delegation.rewardAccounts$))[0].keyStatus).toBe(StakeKeyStatus.Unregistered);
+    });
+
+    it('can delegate vote', async () => {
+      const voteDelegCert: Cardano.VoteDelegationCertificate = {
+        __typename: CertificateType.VoteDelegation,
+        dRep: dRepCredential,
+        stakeCredential
+      };
+      const signedTx = await wallet
+        .createTxBuilder()
+        .customize(({ txBody }) => ({ ...txBody, certificates: [voteDelegCert] }))
+        .build()
+        .sign();
+      const [, confirmedTx] = await submitAndConfirm(wallet, signedTx.tx, 1);
+
+      assertTxHasCertificate(confirmedTx, voteDelegCert);
+      await assertWalletIsDelegating();
+    });
+
+    it('can delegate stake and vote using combo certificate', async () => {
+      const stakeVoteDelegCert: Cardano.StakeVoteDelegationCertificate = {
+        __typename: CertificateType.StakeVoteDelegation,
+        dRep: dRepCredential,
+        poolId,
+        stakeCredential
+      };
+      const signedTx = await wallet
+        .createTxBuilder()
+        .customize(({ txBody }) => ({ ...txBody, certificates: [stakeVoteDelegCert] }))
+        .build()
+        .sign();
+      const [, confirmedTx] = await submitAndConfirm(wallet, signedTx.tx, 1);
+
+      assertTxHasCertificate(confirmedTx, stakeVoteDelegCert);
+      await assertWalletIsDelegating();
+    });
+  });
+
+  describe('with proposal procedure', () => {
+    let actionId: Cardano.GovernanceActionId;
+
+    beforeAll(async () => {
+      const proposalProcedure: Cardano.ProposalProcedure = {
+        anchor,
+        deposit: governanceActionDeposit,
+        governanceAction: { __typename: GovernanceActionType.info_action },
```

Comment:
Do we plan on covering submitting the rest of the actions types?

Reply:
Many of them require a Constitutional Commette or an already approved Proposal Procedure... maybe we can add them later, when we have the required scenario...
On the fly I would say we can add only Parameter Change Proposal... let me take a look